cockpit/IPC/src/Components/Table/Table.tsx

```diff
@@ -406,6 +419,21 @@ export default function CustomTable({
     attributeName="Process"
     activeAttributeMenu={activeAttributeMenu}
     innerRef={attributeRefs.Process}
+  />,
+    },
+    {
+      key: 'Connection',
+      chips: connectionSelection,
+      categoryName: 'Connection',
+      setFiltered: setConnectionSelection,
+      component:
+  <MultiSelectAttribute
+    items={['Connected Only', 'Not Connected Only']}
```

Comment:
```suggestion
    items={['Is connected', 'Not Connected']}
```

Reply:
"Is connected" looks weird in the dropdown.

- Connected 
- Not Connected

opinions @omarhogni ?